habitat_baselines/utils/common.py

```diff
@@ -414,21 +414,15 @@ def img_bytes_2_np_array(
     return (*x[0:3], np.array(images, dtype=np.float32))
 
 
-def create_tar_archive(archive_path: str, dataset_path: str) -> int:
+def create_tar_archive(archive_path: str, dataset_path: str) -> None:
     """Creates tar archive of dataset and returns status code.
     Used in VQA trainer's webdataset.
     """
-    logger.info("[ Creating tar archive .. ]")
-    cmd = [
-        "tar",
-        "--sort",
-        "name",
-        "-cf",
-        archive_path,
-        dataset_path,
-    ]
-    process = subprocess.run(cmd)
-    return process.returncode
+    logger.info("[ Creating tar archive. This will take a few minutes. ]")
+
+    with tarfile.open(archive_path, "w:gz") as tar:
```

Comment:
```suggestion
    with tarfile.open(archive_path, "w") as tar:
```
This is going to apply gzip compression, which is not exactly equivalent to the code before: https://docs.python.org/3/library/tarfile.html

Reply:
@Skylion007 WebDataset seems to be able to handle gzip compressed archives too. Plus, the dataset takes less space now. I don't see why this needs to be changed.

Reply:
Just the extension is tar.gz. Should be good for now I suppose. :)